asv_bench/benchmarks/dataarray_missing.py

```diff
@@ -60,10 +52,11 @@ def setup(self, shape, chunks, limit):
         ),
     )
     def time_ffill(self, shape, chunks, limit):
-        actual = self.da.ffill(dim="time", limit=limit)
```

Comment:
this should not be needed since ffill only ever works with bottleneck and it is installed.

Reply:
Done.